drivers/ipvlan/ipvlan.go

```diff
@@ -0,0 +1,85 @@
+package ipvlan
+
+import (
+	"net"
+	"sync"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/libnetwork/datastore"
+	"github.com/docker/libnetwork/discoverapi"
+	"github.com/docker/libnetwork/driverapi"
+	"github.com/docker/libnetwork/osl"
+)
+
+const (
+	vethLen             = 7
+	containerVethPrefix = "eth"
+	vethPrefix          = "veth"
+	ipvlanType          = "ipvlan"     // driver type name
+	modeL2              = "l2"         // ipvlan mode l2 is the default
+	modeL3              = "l3"         // ipvlan L3 mode
+	hostIfaceOpt        = "host_iface" // host interface -o UX key
```

Comment:
I am not sure if it still stands, other can comment: Shouldn't the driver specific options still follow the `com.docker.network.<driver name>.<option name>` format ?